lib/web_ui/dev/goldens.dart

```diff
@@ -186,7 +186,7 @@ class ImageDiff {
 // Returns text explaining pixel difference rate.
 String getPrintableDiffFilesInfo(double diffRate, double maxRate) =>
   '(${((diffRate) * 100).toStringAsFixed(4)}% of pixels were different. '
-  'Maximum allowed rate is: ${(maxRate * 100).toStringAsFixed(4)}%).';
+  'Maximum allowed rate is: ${((maxRate * 100) - 1).toStringAsFixed(4)}%).';
```

Comment:
I'm not sure why the `- 1` here.

Reply:
Reverted. 
There is this code
if (operatingSystem == OperatingSystem.macOs) {
      maxDiffRatePercent += 1.0;
 }
that causes it to report 1% above on fail.